anchor-reference-server/src/main/java/org/stellar/anchor/reference/controller/UniqueAddressController.java

```diff
@@ -0,0 +1,26 @@
+package org.stellar.anchor.reference.controller;
+
+import com.google.gson.Gson;
+import org.springframework.web.bind.annotation.*;
+import org.stellar.anchor.api.callback.GetUniqueAddressResponse;
+import org.stellar.anchor.reference.service.UniqueAddressService;
+import org.stellar.anchor.util.GsonUtils;
+
+@RestController
+public class UniqueAddressController {
+  final UniqueAddressService uniqueAddressService;
+  static final Gson gson = GsonUtils.builder().create();
+
+  public UniqueAddressController(UniqueAddressService uniqueAddressService) {
+    this.uniqueAddressService = uniqueAddressService;
+  }
+
+  @RequestMapping(
+      value = "/unique_address",
+      method = {RequestMethod.GET})
+  @ResponseBody
```

Comment:
Do we need `@CrossOrigin(origins = "*")` here?

Reply:
We do not need `@CrossOrigin` because this is in reference server.